tests/amazon/aws/triggers/test_emr.py

```diff
@@ -171,6 +171,26 @@ async def test_emr_container_sensors_trigger_exception(mock_query_status):
     assert TriggerEvent({"status": "error", "message": "Test exception"}) in task
 
 
+@pytest.mark.asyncio
+@mock.patch("astronomer.providers.amazon.aws.hooks.emr.EmrContainerHookAsync.check_job_status")
+async def test_emr_container_sensor_trigger_timeout(mock_query_status):
+    """Assert EmrContainerOperatorTrigger max_tries exceed"""
```

Comment:
Please correct the docstrings 
```suggestion
    """Asserts that the  EmrContainerSensorTrigger triggers correct event in case of timeout"""
```

Reply:
fixed